pkg/console/operator/sync_v400.go

```diff
@@ -106,12 +108,21 @@ func sync_v400(co *consoleOperator, originalOperatorConfig *operatorv1.Console,
 	// but we will handle the state of the operand below
 	co.ConditionResourceSyncSuccess(operatorConfig)
 	// the operand is in a transitional state if any of the above resources changed
-	// or if we have not settled on the desired number of replicas
-	if toUpdate || actualDeployment.Status.ReadyReplicas != deploymentsub.ConsoleReplicas {
+	// or if we have not settled on the desired number of replicas or deployment is not uptodate.
+	if toUpdate {
 		co.ConditionResourceSyncProgressing(operatorConfig, "Changes made during sync updates, additional sync expected.")
 	} else {
-		co.ConditionResourceSyncNotProgressing(operatorConfig)
+		version := os.Getenv("RELEASE_VERSION")
+		if !deploymentsub.IsAvailableAndUpdated(actualDeployment) {
+			co.ConditionResourceSyncProgressing(operatorConfig, fmt.Sprintf("Moving to version %s", strings.Split(version, "-")[0]))
```

Comment:
Here and below I wasn't sure if we want to use the `ConditionResourceSyncProgressing()` method which sets the *reason* to "SyncLoopProgressing", but it makes sense to set a single *reason* I guess.